source/casts/index.ts

```diff
@@ -307,6 +360,24 @@ export class Casts {
 		return all;
 	};
 
+	private postCastFilter = (casts, filter: ICastFilter): Array<ICastRow> => {
+		const all = [];
+		for (const cast of casts) {
+			if (filter.time !== undefined) {
+				const castDate = new Date(
```

Comment:
Isn't `cast.value.date` a timestamp anyway? Why do you have to parse it like this?